src/components/Join/JoinForm.tsx

```diff
@@ -0,0 +1,131 @@
+import { useState } from 'react';
+import Input from '../Input';
+
+const JoinForm = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [passwordConfirm, setPasswordConfirm] = useState('');
+  const [nickname, setNickname] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+
+  const [emailError, setEmailError] = useState<string | null>(null);
+  const [passwordError, setPasswordError] = useState<string | null>(null);
+  const [passwordConfirmError, setPasswordConfirmError] = useState<string | null>(null);
+  const [nicknameError, setNicknameError] = useState<string | null>(null);
+  const [phoneNumberError, setPhoneNumberError] = useState<string | null>(null);
```

Comment:
상태변수가 많은데 객체로 묶는건 어떤가요?
```suggestion
  const [form, setForm] = useState<Form>({
    email: "",
    password: "",
    passwordConfirm: "",
    nickname: "",
    phoneNumber: "",
  });

```

Reply:
각각 객체로 묶었습니다. 좋은 방식이네요!